pkg/infrastructure/services/kafka.go

```diff
@@ -29,7 +30,7 @@ const (
 	QuarkusBootstrapAppProp = "kafka.bootstrap.servers"
 
 	// SpringBootstrapAppProp spring boot application property for setting kafka server
-	SpringBootstrapAppProp = "spring.kafka.bootstrap-servers"
+	SpringBootstrapAppProp = "kafka.bootstrap-servers"
```

Comment:
Out of curiosity, why is "kafka.bootstrap-servers" for spring boot and "kafka.bootstrap.servers" for quarkus? Is this defined in the kogito examples or is set by the operator? I guess we should set the same parameter in both.

Reply:
This property depends on the runtime being used. @radtriste are you removing the `spring` prefix because won't work on this runtime? The correct property is `kafka.boostrap-servers`? I remember seeing the Spring Docs `spring.kafka.boostrap-servers`. But I assume you ran some tests and that worked. :p

Reply:
the correct property is `spring.kafka.boostrap-servers`
Thing is that we introduce some specific behavior in our examples. I am correcting them :)